docs/overview.md

```diff
@@ -0,0 +1,844 @@
+FROM Sales;
+```
+
+##### CASTING TO/FROM DATE, TIMESTAMP
+
+```sql
+SELECT CAST(EXTRACT(YEAR FROM saleDate) AS integer), saleDate
+FROM Sales;
+
+SELECT CAST('1975-01-23' AS TIMESTAMP) AS Birthdate, saleDate
+FROM Sales;
+```
+
+Note: MongoDB supports only the TIMESTAMP type.
+
+##### CURRENT_TIMESTAMP
+
+Get the current timestamp:
+
+```sql
+SELECT CURRENT_TIMESTAMP
+FROM Sales;
+```
+
+##### ISO_WEEKDAY
+
+Get the ISO day of the week:
+
+```sql
+SELECT EXTRACT(ISO_WEEKDAY FROM saleDate), saleDate
+FROM Sales;
+```
+
+#### Numeric Functions
+
+##### TO/FROM EPOCH
+
+To Epoch:
+
+```sql
+SELECT CAST(saleDate AS LONG)
+FROM Sales;
+```
+
+From Epoch:
+
+```sql
+SELECT CAST(epochValue AS TIMESTAMP)
+FROM SomeTable;
+```
+
+#### Unsupported Functions
+
+- SIMILAR TO
+- RANDOM
+- Timezone Conversion
+Not Supported: MongoDB stores dates in UTC.
+- GROUP_CONCAT
+
+### Additional Notes
+
+- Polymorphic Schemas:
+Be cautious when using aggregate functions on fields with polymorphic schemas or non-comparable types like documents and arrays.
+The term "polymorphic" schema is used to refer to a field/column that can have multiple types, e.g. int and string.
+
+- Escape Characters:
+Use the ESCAPE clause to specify custom escape characters in LIKE patterns.
+
+- Alias Assignments:
+Required when using nested selects or derived tables.
+
+- Date Functions:
+MongoSQL supports various date functions, but only the TIMESTAMP data type is available.
+
+## Additional Features
+
+### Security Features
+
+The MongoDB Atlas SQL ODBC driver supports all authentication mechanisms
+supported by MongoDB (x509, OAuth, LDAP, etc...). See [authentication mechanisms](https://www.mongodb.com/docs/manual/core/authentication/#authentication-mechanisms)
+for a full list of supported authentication mechanisms.
+
+If [configured](https://www.mongodb.com/docs/manual/tutorial/configure-ssl/), the MongoDB Atlas SQL ODBC driver supports TLS/SSL connections.
+
+### Logging and Diagnostics
+
+By default, the MongoDB Atlas SQL ODBC driver produces logs in `/%HOME%/MongoDB/Atlas SQL/logs`.
+
+Logging can be fine-tuned by passing the `LOGLEVEL` property in your ODBC connection
+string or configuring it in your DSN.
+
+The following is a list of valid values for LOGLEVEL and their precedence:
+- ERROR - Only errors will be logged.
+- WARN - Information about operations that could be an error in future versions of the driver.
+- INFO - Informational log messages. Default
+- DEBUG - Debug information useful for debegging purposes. Enable this mode to submit a log with a HELP ticket.
+- TRACE - Extremely verbose logging, including network traffic and information from ancillary libraries used in the driver. Not recommended unless
+asked for by MongoDB support.
+
+Each LOGLEVEL will include log messages emitted at a higher precedence. For example, INFO will include all INFO, WARN, and ERROR
+messages, while ERROR will include only ERROR messages.
+
+## Troubleshooting
+
+### Common Issues
```

Comment:
We can add this message:

> Enterprise edition detected, but mongosqltranslate library not found.

Ensure that the `mongosqltranslate` library exists in the same directory as the MongoDB SQL ODBC driver.